### PR TITLE
Improved loading of the Website

### PR DESCRIPTION
**Pull request by parasss19**
## Description 📝

- Fixes/Closes: #485
- Earlier the images are in png/jpg format which requires more space and loading time
- I go through each and every folder and changed all images to webp without breaking any image

## Changes 💬

- High-level overview of the changes introduced.
- Step-by-step breakdown of the implementation, if applicable.

### 1 Original
Screenshot 2023-07-16 194812

### 2 Updated
time loading

### CODE REVIEW

File: package-lock.json
Comment (PiyushKalyanpy) [inline package-lock.json:1]: Please remove this file from PR, refer #466

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

**Comment by parasss19** [comment]
Hey @PiyushKalyanpy I removed the package-lock.json file 
- Please provide level2 or level3 as it require changes in so many files
- And also i have to go through each and every files to change around 150+ images

**Comment by PiyushKalyanpy** [comment]
I will add the label once I merge the pull request.

**Comment by PiyushKalyanpy** [comment]
> Hey @PiyushKalyanpy I removed the package-lock.json file 
> - Please provide level2 or level3 as it require changes in so many files
> - And also i have to go through each and every files to change around 150+ images

Please create a new pr without that file, your pr should not look like (more than 8000 deletions) you can refer the discussion tab

**Pull request closed without merging**